stack_master.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "table_print"
   spec.add_dependency "dotgpg"
   spec.add_dependency "deep_merge"
+  spec.add_dependency "cfndsl"
```

Comment:
Cool. We'll also need to update the readme about the fact that they need to be installed separately...